### ISSUE TITLE
Multi-touch point contact problem

### ISSUE THREAD

**Issue by IvanLu-lyf** [MartinKotoucSourek/openHFDIB-DEM#2]
I want to pimpleHFDIBFoam to simulate a non-Convex particle driven by pipe flow in a straight pipe, this particle may have multi-touch points with the pipe. Can this solver handle this kind of problem?

**Comment by MartinKotoucSourek** [comment]
Yes, the solver can handle such a problem. Moreover, the "multi-contact" case will be extensively improved in the upcoming version 2.3.

**Comment by IvanLu-lyf** [comment]
When I used the solver, I encountered a non-physical phenomenon of complex geometry penetrating the pipe, what could be the reason for this?

**Comment by IvanLu-lyf** [comment]
I'm sorry for replying to you late. Do you have any other information on DEM setup besides the user guide?

**Comment by IvanLu-lyf** [comment]
Does this algorithm realize particle deformation through soft-DEM? Or is soft-DEM just involved in particle-particle and particle-wall contact force calculations?

**Comment by MartinIsoz** [comment]
Hello, thank you for the continued interest in our code. The term "soft-DEM" is used to represent the fact that an OVERLAP between two otherwise RIGID solids is used to estimate the contact forces in both the particle-wall and particle-particle contact. At the moment, no deformations of the solids are taken into account.
